services/src/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/InstallIngressRuleCommand.java

```diff
@@ -60,6 +60,8 @@
 @Getter
 public class InstallIngressRuleCommand extends InstallTransitRuleCommand {
 
+    private static final int DEFAULT_FLOW_PRIORITY = FLOW_PRIORITY - 1;
```

Comment:
let's not duplicate these constants, make them public in switch manager if needed

Reply:
+1

Reply:
honestly, I don't like the idea to keep such constants in switch manager. We can rather place it somewhere else or come up with another solution. Let's do it in the scope of another ticket/PR?

Reply:
+1 for somewhere else, but I really don't like the same constants in several places.